mlflow/pyfunc/__init__.py

```diff
@@ -653,6 +687,9 @@ def load_model(model_uri: str, suppress_warnings: bool = True, dst_path: str = N
                      path will be created.
     """
     local_path = _download_artifact_from_uri(artifact_uri=model_uri, output_path=dst_path)
+
+    check_requirements_and_local_installed_mismatch(local_path)
```

Comment:
Do we add `check_requirements_and_local_installed_mismatch` to all flavors?

Reply:
@dbczumar Do we need this ?